src/lib/goToLocationSettings.js

```diff
@@ -10,30 +9,18 @@ export default function goToLocationSettings() {
 
   const userAgent = getUserAgent();
 
-  if (isCordova()) {
-    if (
-      userAgent.os.name === 'Android' &&
-      typeof window !== 'undefined' &&
-      window.cordova &&
-      window.cordova.diagnostics &&
-      typeof window.cordova.diagnostics.switchToLocationSettings === 'function'
-    ) {
-      window.cordova.diagnostics.switchToLocationSettings();
```

Comment:
Hmm, sure we don't need to keep this as location handling is done natively?

Reply:
This will be done by the native apps, they will hook into the urls used here

Reply:
iOS is already done, https://github.com/sozialhelden/wheelmap-native-wrapper/commit/876dc653bdfe321983cd0fd7f6d0e6f93575e847